test/adapters/redis_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+require 'flipper/adapters/redis'
+
+class RedisTest < MiniTest::Test
+  prepend SharedAdapterTests
+
+  def setup
+   options = {}
+   if ENV['BOXEN_REDIS_URL']
```

Comment:
I actually don't use boxen anymore so I'm ok with leaving this part out if you want. We can also leave it and just do a remove boxen artifacts PR later. Either way is fine by me!